lib/brainstem/concerns/controller_dsl.rb

```diff
@@ -65,10 +65,20 @@ def documented!
         # whereas setting it within an action context will force that action to
         # be undocumented.
         #
-        def nodoc!
+        def nodoc!(_description = nil)
           configuration[brainstem_params_context][:nodoc] = true
```

Comment:
Why are we not saving the description?

Reply:
we don't currently have a use for it. In the future, when we actually use the description somewhere, we can save the data and use it, but right now, there isn't any reason to. It's just setting us up for a future where we'll have this data for whatever use we decide is appropriate.